Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -33,6 +33,8 @@ STATIC CHAR16 mPowerFailureStateEfiVar[] = L"PowerFailureState";
 STATIC CHAR16 mResizeableBarsEnabledEfiVar[] = L"PCIeResizeableBarsEnabled";
 STATIC CHAR16 mOptionRomPolicyEfiVar[] = L"OptionRomPolicy";
 STATIC CHAR16 mEnableCameraEfiVar[] = L"EnableCamera";
+STATIC CHAR16 mBatteryStartThresholdEfiVar[] = L"BatteryStartThreshold";
+STATIC CHAR16 mBatteryStopThresholdEfiVar[] = L"BatteryStopThreshold";

```

Comment:
Can't it be just one variable of struct type? Similar to how watchdog is done. Then we will have less access to variable store

Reply:
Right, combined both thresholds to a single EFI var.